CONTRIBUTING.md

```diff
@@ -48,7 +48,7 @@ If you wish to build the C++ portions of the documentation, you should also have
 
 ## Builds
 
-We use [Gnu make](https://www.gnu.org/software/make/) for builds.  If you have installed [CmdStan](https://mc-stan.org/users/interfaces/cmdstan) as required for the source to work, then you will already have Gnu make and a sufficient C++ compiler.
+We use [Gnu make](https://www.gnu.org/software/make/) for builds.  If you have installed [CmdStan](https://mc-stan.org/users/interfaces/cmdstan), then you will already have Gnu make and a sufficient C++ compiler.
```

Comment:
Maybe this is a silly concern, but can we change this sentence to be more clear that a user needs to be able to build, more than just clone and `make stan-update`, a Stan program using CmdStan for BridgeStan to work correctly?